rasa/shared/importers/rasa.py

```diff
@@ -40,11 +39,16 @@ def __init__(
             training_data_paths, YAMLStoryReader.is_test_stories_file
         )
 
-        self.config = autoconfig.get_configuration(config_file, training_type)
+        self.config_file = config_file
 
     def get_config(self) -> Dict:
         """Retrieves model config (see parent class for full docstring)."""
-        return self.config
+        if not self.config_file or not os.path.exists(self.config_file):
+            logger.debug("No configuration file was provided to the RasaFileImporter.")
+            return {}
+
+        config = rasa.shared.utils.io.read_model_configuration(self.config_file)
```

Comment:
Do we risk using the non-autoconfigured config sometimes?

Reply:
Recipes do auto-configure work now, so importers always read raw files. It is recipe's responsibility to auto-fill now (and dump the file after auto-fill), so at this point it is just raw config file read, and we can get what recipe needs to be extracted afterwards.